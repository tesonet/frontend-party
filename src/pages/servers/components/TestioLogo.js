import React from 'react';

export const TestioLogo = ({ tableColumns, handleSorting }) => {
    return (
    	<div>
    		Logo
    	</div>
    )
}