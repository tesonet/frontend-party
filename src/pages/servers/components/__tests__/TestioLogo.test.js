import { TestioLogo } from '../TestioLogo';

it("should match snapshot", () => {
    const wrapper = shallow(
        <TestioLogo logo='logo'/>
    );
    expect(wrapper).toMatchSnapshot();
});