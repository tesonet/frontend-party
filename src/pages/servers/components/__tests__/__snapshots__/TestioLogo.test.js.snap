// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="testio-logo"
>
  <img
    alt="testio"
    src="logo"
  />
</div>
`;
